File: solutions/1603-running-sum-of-1d-array/solution.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    let adder=0;
    for(let i=0;i<nums.length;i++)
    {
        adder+=nums[i];
        nums[i]=adder;
    }
    return nums;
};

File: solutions/1603-running-sum-of-1d-array/solution.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        adder=0;
        for i in range(len(nums)):
            adder+=nums[i];
            nums[i]=adder;
        return nums;
